synapse/tests/tlds-alpha-by-domain.txt

```diff
@@ -0,0 +1,1543 @@
+# Version 2017101000, Last Updated Tue Oct 10 07:07:01 2017 UTC
```

Comment:
I think we can just include a test file, since the test is just testing that openfile will go out and download a file